test/transmuxer/transmuxer_integration.js

```diff
@@ -305,10 +305,10 @@ filterDescribe('Transmuxer Player', checkNoBrokenEdge, () => {
 
     it('H.265 in TS', async () => {
       const chromeVersion = shaka.util.Platform.chromeVersion();
-      if (shaka.util.Platform.isWindows() &&
-          chromeVersion && chromeVersion === 117) {
-        // It appears that Chrome 117 beta in Windows is incorrectly reporting
-        // H.265 in MediaCapabilities
+      if (shaka.util.Platform.isWindows() && chromeVersion) {
+        // It appears that Chrome 122 in Windows is still incorrectly reporting
```

Comment:
Has it started from version 117 or it it was detected on version 117?

If the former, maybe adjust the comment with "starting from Chrome 117 ..."

Reply:
I don't know.  I didn't add the original AFAIK...

In my opinion, with an evergreen browser, the most important thing is the most recent version where somebody checked and found that there was still an issue.